src/components/Button/Button.styled.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+import { colors } from "../../global/colors";
+
+
+export const BtnWrapper = styled.a`
+    border-radius: 4px;
+    text-transform: uppercase;
+    background-color: ${colors.primary};
+    border: none;
+    padding: 8px 20px;
+    margin: 4px 2px;
+    color: ${colors.bgColor};
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    transition-duration: 0.4s;
+    cursor: pointer;
+    float: right;
+
+    &.primary-color-btn {
```

Comment:
With styled components it would be better to use props instead classes to define primary and default button colors

Reply:
Done